core/src/main/java/com/linecorp/armeria/common/NonWrappingRequestContext.java

```diff
@@ -54,6 +55,7 @@
     private volatile HttpRequest req;
     @Nullable
     private volatile RpcRequest rpcReq;
+    protected UUID uuid;
```

Comment:
- `private final` unless there is a good reason to make it mutable and protected.
- I guess we need to move this field right before `method`, because it is an essential non-null per-request field.